releasenotes-dca/notes/add-exponential-backoff-to-syncinformer-9385d768f8ca21ce.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add exponential backoff for syncing with the kubernetes informers. The max total retry period and max retry delay can be configured with kube_cache_sync_total_retry_seconds, kube_cache_sync_max_retry_delay_seconds.
```

Comment:
You updated your PR to only be able to setup the `kube_cache_sync_max_retry_delay_seconds`, I think you have to update your release note here to reflect that? :)

Reply:
Nice catch 👍